src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,218 @@
+                car = parseCar(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car by id " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getAllDriversFromCar(car.getId()));
+        }
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String getAll = "SELECT c.id as car_id, manufacturer_id, model, "
+                + "m.name, m.country FROM cars c JOIN manufacturers m "
+                + "ON c.manufacturer_id = m.id WHERE c.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllStatement
+                        = connection.prepareStatement(getAll)) {
+            ResultSet resultSet = getAllStatement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(parseCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get a list of cars "
+                    + "from cars table. ", e);
+        }
+        for (Car car : cars) {
+            car.setDrivers(getAllDriversFromCar(car.getId()));
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String updateRequest = "UPDATE cars SET model = ?, manufacturer_id = ? "
+                + "WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateStatement
+                        = connection.prepareStatement(updateRequest)) {
+            updateStatement.setString(1, car.getModel());
+            updateStatement.setLong(2, car.getManufacturer().getId());
+            updateStatement.setLong(3, car.getId());
+            updateStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't update "
+                    + car + " in driversDB.", e);
+        }
+        deleteAllDriversFromCar(car);
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteRequest = "UPDATE cars SET is_deleted = TRUE WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteStatement
+                        = connection.prepareStatement(deleteRequest)) {
+            deleteStatement.setLong(1, id);
+            return deleteStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete a car by id " + id + " ", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String getAllByDriverRequest = "SELECT c.id as car_id, manufacturer_id, model, "
+                + "name , country FROM cars c "
```

Comment:
```suggestion
        String getAllByDriverRequest = "SELECT c.id as car_id, manufacturer_id, model, "
                + "name , country " 
                + "FROM cars c "
```
this way it loocs nicer)